tubemq-client-twins/tubemq-client-go/client/consumer_impl.go

```diff
@@ -744,3 +752,15 @@ func (c *consumer) closeAllBrokers() {
 	}
 	log.Infof("[CONSUMER] closeAllBrokers end, client=%s", c.clientID)
 }
+
+// Invoke implementation of TubeMQ consumer.
+func (c *consumer) Invoke(config *config.Config, opts ...config.Option) (Consumer, error) {
```

Comment:
Maybe you can use  NewConfig(opts ..config.Option) *config.Config is a good idea.
And use NewConsumer(config *config.Config) to start.